hummingbot/core/utils/async_utils.py

```diff
@@ -0,0 +1,16 @@
+import asyncio
+import logging
+
+
+def asyncio_ensure_future(*args, **kwargs):
```

Comment:
The name should reflect the purpose of the function. I'd recommend naming it `safe_ensure_future()` instead.